op-bindings/Makefile

```diff
@@ -12,7 +12,7 @@ version:
 
 compile:
 	cd $(contracts-dir) && \
-		pnpm build
+		forge build ${FORGE_BUILD_ARGS}
```

Comment:
I'm not sure if there are any side effects. The reason I added this is because there were issues with `pnpm` results being cached by `nx` and also `forge` not invalidating the cache when it's running in the docker container.

Reply:
It shouldn't have side effects other than that `pnpm build` is only doing `forge build`.